apps/dcos_dns/src/dcos_dns_handler.erl

```diff
@@ -337,3 +355,25 @@ max_payload_size(
     PayloadSize;
 max_payload_size(_DNSMessage) ->
     ?MAX_PACKET_SIZE.
+
+%%%===================================================================
+%%% Metrics functions
+%%%===================================================================
+
+-spec(init_metrics() -> ok).
+init_metrics() ->
+    prometheus_counter:new([{name, dns_forwarder_failures_total},
```

Comment:
Please use 4 spaces for indentation, it's hard to force this code style in the project when there is even one person who doesn't use emacs / fancy plugins.

```
prometheus_counter:new([
    {name, dns_forwarder_failures_total},
    …
]),
```

Reply:
Ups, I forgot about this one on my notes from our last meeting. I will see if I can configure `emacs`  so it does not do the fancy formatting.